### PR TITLE
Add kubectl-point

### PR DESCRIPTION
**Pull request by davidchua**
Creates a headless service to an external source with optional ingress

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/point.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  # 'name' must match the filename of the manifest. The name defines how
```
Comment (ahmetb) [inline plugins/point.yaml:4]: Please remove these comments.

**Review by ahmetb — CHANGES_REQUESTED** [review]

File: plugins/point.yaml (hunk 1 above)
Comment (davidchua) [reply plugins/point.yaml]: Yikes! Thanks! I've removed them

File: plugins/point.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: point
+spec:
+  version: "v0.1.0"
+  homepage: https://github.com/davidchua/kubectl-point
+  shortDescription: "Creates a headless service to an external source with optional ingress"
```
Comment (ahmetb) [inline plugins/point.yaml:8]: Just fyi this probably won't fit where it's shown and will be truncated. Limit to 50 chars if you can.

File: plugins/point.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  description: |
+    Generates and creates a headless service and/or ingress to point to any external source
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: arm64
+    uri: https://github.com/davidchua/kubectl-point/releases/download/v0.1.0/kubectl-point_0.1.0_Darwin_arm64.tar.gz
+    sha256: 4e9f6668207c4227181f98aae447229677941446e670abcca01c6ee5abc65e8b
+    files:
```
Comment (ahmetb) [inline plugins/point.yaml:18]: This field is redundant if you do *,. like you do here.

File: plugins/point.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: point
+spec:
+  version: "v0.1.0"
+  homepage: https://github.com/davidchua/kubectl-point
+  shortDescription: "Creates a headless service to an external source with optional ingress"
+  description: |
+    Generates and creates a headless service and/or ingress to point to any external source
```
Comment (ahmetb) [inline plugins/point.yaml:10]: Add a trailing period and maybe go into just a little bit more into detail as to what the feature list is etc.

**Comment by davidchua** [comment]
Thanks for the comments! Appreciate the time taken to review this!

Made the changes as requested. I had to take a bit more time to really think hard about how to make the description a bit more concise. I hope I got it!

**Comment by ahmetb** [comment]
Sorry this entirely fell off my radar. Any problems with going ahead and accepting this?

**Pull request closed without merging**